### ISSUE TITLE
Ps2 controller doesnt work when you open a game in ubuntu 20.04.

### ISSUE THREAD

**Issue by JustARandomUser550** [PCSX2/pcsx2#4470]
**Describe the bug**
When you open the ps2 bios, controller works fine. but when you open a game (iso) then the controller doesnt respond anymore) (Happens on all games)
 Im using a usb to ps2 controller adapter. This works fine for windows but not sure for why for ubuntu

**To Reproduce**
connect the ps2 (using an adapter) to pc
open up pcsx2
open up any game
controller doesnt respond
open up ps2 bios instead
controller works fine

**Expected behavior**
Controller should function and respond to the game. 

**GS Settings**
No changes to the settings. Its a fresh compiled pcsx2 with the latest build with the default settings

**Emulation Settings**
No changes to the settings. Its a fresh compiled pcsx2 with the latest build with the default settings

**System Info (please complete the following information):**
 -   PCSX2 Revision: dev:1280
 -   OS: Ubuntu 20.04
 -   CPU: i7-9700k
 -   GPU: RTX 2060 super

**Logs and Dumps**
[logs.txt](https://github.com/PCSX2/pcsx2/files/6566818/logs.txt)

**Comment by F0bes** [comment]
Would you be able to test the latest dev build. If it doesn't work there can you give an updated log? Thanks.

**Comment by Raul-Prisecaru** [comment]
Hi, just tried it and still doesn't seem to work. I have tried V1.7.2113 and still seems to be the same. 


Going to gamepad settings, I went to "gamepad configuration" and in the list box (all I see is two of: "twin USB Joystick") to choose the controller, I switched between them and enabled rumble to make sure it's being selected correctly and detected. Unfortunately doesn't seem to work for any of the 2 options but 

When I go to "gamepad 1" and go to "gamepad configuration", both options seems to work just fine (both of the options rumbles).
I don't know why "gamepad1" seems to work but I'm assuming is because Linux is detecting my controller as player 2 for some reason. Tried going through all of my USB ports and both changed. 

Tried it with ubuntu 20.04 and pop os 21.04 and same specs as mentioned on my first post.
Hope this helps in some way
